src/runtime/ndarray.cc

```diff
@@ -185,6 +185,38 @@ NDArray NDArray::FromDLPack(DLManagedTensor* tensor) {
   return NDArray(GetObjectPtr<Object>(data));
 }
 
+void NDArray::CopyToBytes(void* data, size_t nbytes) const {
+  CHECK(data != nullptr);
```

Comment:
Last change: perhaps it is better to bring the common impl in this function, and TVMArrayCopyToBytes to a common function
```
void ArrayCopyToBytes(DLTensor* handle, void* data, size_t nbytes).
```
and have both functions call into this common function to avoid impl duplications.

Reply:
I have ArrayCopyFromBytes/ArrayCopyToBytes as the single implementation for the C and C++ API.
Let me know if I missed anything.